test/lambda-handler-tests.js

```diff
@@ -277,6 +277,22 @@ describe('handler', function() {
 				});
 		});
 
+		it('next() call is resolved with result value', function() {
+			const obj = { foo: 'bar' };
+			let nextResult;
+
+			const fixture = lambdaHandler()
+				.use(async (event, context, next) => {
+					nextResult = await next();
```

Comment:
I don't know any better way to assert this. Ideas?

Reply:
this is fine to me, no need to get all hi-fi

however, what about a test with chained next() that overwrites the result, confirming that expected is returned?